src/initialization.js

```diff
@@ -25,16 +27,48 @@ var initialization = {
                 };
             } else {
                 isInitialized = true;
-                loadEventsAndPages();
+                loadWebXEventsAndPages();
             }
+
+            if (!window.optimizelyClientInstance && settings.useFullStack) {
+                var errorHandler = function() {};
+
+                if (customFlags && customFlags['OptimizelyFullStack.ErrorHandler']) {
+                    errorHandler = customFlags['OptimizelyFullStack.ErrorHandler'];
+                }
+
+                var instantiateFSClient = function() {
+                    var optimizelyClientInstance = window.optimizelySdk.createInstance({
```

Comment:
this needs to be set as `window.optimizelyClientInstance` so that it is accessible in other modules